{
  "type": "issue",
  "issue": {
    "id": 14438635,
    "node_id": "MDU6SXNzdWUxNDQzODYzNQ==",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/2659",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/2659/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/2659/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/2659/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/2659",
    "number": 2659,
    "state": "closed",
    "state_reason": "completed",
    "title": "Assertion failed on checkqueue.h line 185",
    "body": "![bitcoin error](https://f.cloud.github.com/assets/1612927/516362/3c71c2cc-beac-11e2-96ed-54fe014701a4.jpg)\n\nI was doing some stress test on my bitcoin-qt client on testnet, so I decided to create about 70000 address and after this I created about 15000 transactions.\n\nThe addresses and the transactions was created ok, but only a few transaction was confirmed. As I was not sure what was happening, I decided to close the program and reopen it rescanning the blockchain to see how the client will behave under this conditions (Used the command \"bitcoin-qt.exe -testnet -txindex -reindex\" ). Well, the gui become too unresponsive (about 10 seconds frozen and then processed the Gui Messages and become unresponsive again for another 10 seconds), but after rescanning about 20000 blocks the above message appear. I closed the program and reopened with the command  \"bitcoin-qt.exe -testnet\" the rescan continued from where it stoped when the crashes happened. But after another 20000 blocks the message appeared again. Well, this message appeared more two times (when was about 12000 and about 11000 blocks left), after this all went well and the reindex finished without more crashes. So, any Ideas why this happenend?\n\nI'm using Windows XP SP3, intel core 2 duo and 4 GB of memory and bitcoin 0.8.1.\n\nRegards\n",
    "user": {
      "login": "LeandroJF",
      "id": 1612927,
      "node_id": "MDQ6VXNlcjE2MTI5Mjc=",
      "avatar_url": "https://avatars.githubusercontent.com/u/1612927?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/LeandroJF",
      "html_url": "https://github.com/LeandroJF",
      "followers_url": "https://api.github.com/users/LeandroJF/followers",
      "following_url": "https://api.github.com/users/LeandroJF/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/LeandroJF/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/LeandroJF/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/LeandroJF/subscriptions",
      "organizations_url": "https://api.github.com/users/LeandroJF/orgs",
      "repos_url": "https://api.github.com/users/LeandroJF/repos",
      "events_url": "https://api.github.com/users/LeandroJF/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/LeandroJF/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [],
    "assignees": [],
    "author_association": "NONE",
    "locked": true,
    "active_lock_reason": "resolved",
    "comments": 2,
    "closed_at": "2013-05-17T18:51:14Z",
    "created_at": "2013-05-17T05:41:35Z",
    "updated_at": "2021-09-08T12:18:29Z"
  },
  "events": [
    {
      "event": "commented",
      "id": 18045651,
      "node_id": "MDEyOklzc3VlQ29tbWVudDE4MDQ1NjUx",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/18045651",
      "actor": {
        "login": "Diapolo",
        "id": 1419649,
        "node_id": "MDQ6VXNlcjE0MTk2NDk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1419649?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Diapolo",
        "html_url": "https://github.com/Diapolo",
        "followers_url": "https://api.github.com/users/Diapolo/followers",
        "following_url": "https://api.github.com/users/Diapolo/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Diapolo/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Diapolo/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Diapolo/subscriptions",
        "organizations_url": "https://api.github.com/users/Diapolo/orgs",
        "repos_url": "https://api.github.com/users/Diapolo/repos",
        "events_url": "https://api.github.com/users/Diapolo/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Diapolo/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2013-05-17T06:20:37Z",
      "updated_at": "2013-05-17T06:20:53Z",
      "author_association": "NONE",
      "body": "Can you please try again with 0.8.2 RC1, which has some thread changes that could be related here.\n",
      "user": {
        "login": "Diapolo",
        "id": 1419649,
        "node_id": "MDQ6VXNlcjE0MTk2NDk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1419649?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Diapolo",
        "html_url": "https://github.com/Diapolo",
        "followers_url": "https://api.github.com/users/Diapolo/followers",
        "following_url": "https://api.github.com/users/Diapolo/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Diapolo/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Diapolo/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Diapolo/subscriptions",
        "organizations_url": "https://api.github.com/users/Diapolo/orgs",
        "repos_url": "https://api.github.com/users/Diapolo/repos",
        "events_url": "https://api.github.com/users/Diapolo/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Diapolo/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/2659#issuecomment-18045651",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/2659"
    },
    {
      "event": "commented",
      "id": 18079520,
      "node_id": "MDEyOklzc3VlQ29tbWVudDE4MDc5NTIw",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/18079520",
      "actor": {
        "login": "LeandroJF",
        "id": 1612927,
        "node_id": "MDQ6VXNlcjE2MTI5Mjc=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1612927?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/LeandroJF",
        "html_url": "https://github.com/LeandroJF",
        "followers_url": "https://api.github.com/users/LeandroJF/followers",
        "following_url": "https://api.github.com/users/LeandroJF/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/LeandroJF/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/LeandroJF/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/LeandroJF/subscriptions",
        "organizations_url": "https://api.github.com/users/LeandroJF/orgs",
        "repos_url": "https://api.github.com/users/LeandroJF/repos",
        "events_url": "https://api.github.com/users/LeandroJF/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/LeandroJF/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2013-05-17T18:51:14Z",
      "updated_at": "2013-05-17T18:51:14Z",
      "author_association": "NONE",
      "body": "After posting the issue here, i went to bed and when I woke up, almost all my transactions was confirmed, so I decided to try to rescan the blockchain again to see if I would receive the error agin, but this time it rescaned the blockchain without problem. I thought \"maybe the error was ocurring when there's too many unconfirmed transactions\". I decided to create more 15000 transactions and then I rescanned the blockchain again. And again no problem during the rescan, all went fine even with more than 15000 transactions to confirm. So as I wasn't able to reproduce the error again, I'm closing this.\n\nPS: all above was done with the same 0.8.1 version.\n",
      "user": {
        "login": "LeandroJF",
        "id": 1612927,
        "node_id": "MDQ6VXNlcjE2MTI5Mjc=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1612927?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/LeandroJF",
        "html_url": "https://github.com/LeandroJF",
        "followers_url": "https://api.github.com/users/LeandroJF/followers",
        "following_url": "https://api.github.com/users/LeandroJF/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/LeandroJF/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/LeandroJF/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/LeandroJF/subscriptions",
        "organizations_url": "https://api.github.com/users/LeandroJF/orgs",
        "repos_url": "https://api.github.com/users/LeandroJF/repos",
        "events_url": "https://api.github.com/users/LeandroJF/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/LeandroJF/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/2659#issuecomment-18079520",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/2659"
    },
    {
      "event": "closed",
      "id": 49274348,
      "node_id": "MDExOkNsb3NlZEV2ZW50NDkyNzQzNDg=",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/49274348",
      "actor": {
        "login": "LeandroJF",
        "id": 1612927,
        "node_id": "MDQ6VXNlcjE2MTI5Mjc=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1612927?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/LeandroJF",
        "html_url": "https://github.com/LeandroJF",
        "followers_url": "https://api.github.com/users/LeandroJF/followers",
        "following_url": "https://api.github.com/users/LeandroJF/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/LeandroJF/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/LeandroJF/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/LeandroJF/subscriptions",
        "organizations_url": "https://api.github.com/users/LeandroJF/orgs",
        "repos_url": "https://api.github.com/users/LeandroJF/repos",
        "events_url": "https://api.github.com/users/LeandroJF/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/LeandroJF/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2013-05-17T18:51:14Z"
    },
    {
      "event": "locked",
      "id": 5271928202,
      "node_id": "LOE_lADOABII584A3FDrzwAAAAE6Oz2K",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/5271928202",
      "actor": {
        "login": "MarcoFalke",
        "id": 6399679,
        "node_id": "MDQ6VXNlcjYzOTk2Nzk=",
        "avatar_url": "https://avatars.githubusercontent.com/u/6399679?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/MarcoFalke",
        "html_url": "https://github.com/MarcoFalke",
        "followers_url": "https://api.github.com/users/MarcoFalke/followers",
        "following_url": "https://api.github.com/users/MarcoFalke/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/MarcoFalke/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/MarcoFalke/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/MarcoFalke/subscriptions",
        "organizations_url": "https://api.github.com/users/MarcoFalke/orgs",
        "repos_url": "https://api.github.com/users/MarcoFalke/repos",
        "events_url": "https://api.github.com/users/MarcoFalke/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/MarcoFalke/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2021-09-08T12:18:29Z",
      "lock_reason": "resolved"
    }
  ]
}